src/material/grid-list/_grid-list-theme.scss

```diff
@@ -1,35 +1,68 @@
+@use 'sass:map';
 @use '../core/theming/theming';
 @use '../core/theming/inspection';
 @use '../core/typography/typography';
 @use '../core/tokens/m2/mat/grid-list' as tokens-mat-grid-list;
 @use '../core/style/sass-utils';
 @use '../core/tokens/token-utils';
 
-@mixin base($theme) {}
+@mixin base($theme) {
+  @if inspection.get-theme-version($theme) == 1 {
+    @include _theme-from-tokens(inspection.get-theme-tokens($theme, base));
+  }
+  @else {}
```

Comment:
nitL should we remove the empty `@else`.

Reply:
ehh, I've been leaving it as a hint if we need to add any M2 tokens, that they belong in the else (I suspect we will add more tokens for some of the components in order to fully express M3, and those will need corresponding M2 values)